attendance/resources/rekognition.py

```diff
@@ -0,0 +1,22 @@
+import boto3
+from flask import current_app
+
+
+def compare_faces(targetId, sourceId):
+    client = boto3.client('rekognition',
+                          aws_access_key_id="test",
+                          aws_secret_access_key="test",
+                          region_name='us-west-2')
```

Comment:
Would these keys work when deployed?

Reply:
As discussed we will bypass this call for now and enable it back once AWS keys are obtained.

Reply:
Yes